include/ginkgo/core/base/mpi.hpp

```diff
@@ -116,8 +116,9 @@ GKO_REGISTER_MPI_TYPE(long long, MPI_LONG_LONG_INT);
 GKO_REGISTER_MPI_TYPE(float, MPI_FLOAT);
 GKO_REGISTER_MPI_TYPE(double, MPI_DOUBLE);
 GKO_REGISTER_MPI_TYPE(long double, MPI_LONG_DOUBLE);
-GKO_REGISTER_MPI_TYPE(std::complex<float>, MPI_CXX_FLOAT_COMPLEX);
-GKO_REGISTER_MPI_TYPE(std::complex<double>, MPI_CXX_DOUBLE_COMPLEX);
+// CXX BINDING is deprecated
```

Comment:
The `MPI_CXX_*_COMPLEX` are not part of the deprecated MPI CXX binding. So perhaps just remove this comment.